generators/entity-server/templates/src/main/java/package/common/get_all_stream_template.ejs

```diff
@@ -19,16 +19,31 @@
 <%
 const mapper = entityInstance  + 'Mapper';
 const entityToDtoReference = mapper + '::' + 'toDto';
+const reactiveSql = (databaseType === 'sql' && reactive);
+let toResponseEntityPrefix = '';
+let toResponseEntityPostfix = '';
+if (pagination !== 'no') {
+    if (reactiveSql) {
+        toResponseEntityPostfix = '.map(ResponseEntity::ok)';
```

Comment:
I've the feeling that it make the template more difficult to understand.
Sometimes a little duplication is not problematic if it makes things more maintainable.

Reply:
Well, I think, adding more repetition doesn't make it more maintainable, or more readable. 
The real problem is that in these templates it's really hard to create and use bigger abstractions - without painful escaping.
I'm thinking about, something like this:
```

function wrapResponse(response) {
   if (pagination !== 'no') {
       if (reactiveSql) {
          return response + ".map(ResponseEntity::ok)";
       } else {
          return "new ResponseEntity<>(" + response + ".map(ResponseEntity::ok)";
       }
   }
   return response;
}

```

and in template, it would be nice, if we could wrote something like:
` <%= wrapResponse( %>....<% ); %>
`
or 
`<%wrapResponse%>.....</%wrapResponse%> `

But AFAIK, this is not possible :disappointed: